src/config/schema-type.ts

```diff
@@ -118,6 +121,13 @@ export const schemaTypes = {
 		type: PermissiveBoolean,
 		default: false,
 	},
+	/**
+	 * The timeout for requests to the balenaCloud api
+	 */
+	apiRequestTimeout: {
+		type: PermissiveNumber,
+		default: 59000,
+	},
```

Comment:
You added the configuration here, but not on `config/schema.ts` which means the value is not really configurable. Given that it should perhaps be set on `lib/constants.ts` which is a simpler change?

Reply:
Alternatively we can make it configurable, perhaps via config.json or a config variable

Reply:
Yeah I'm thinking a config var could be good, not that we intend to use it but I've never been upset in the past when the API is able to modify supervisor behavior because it gives us more options for dealing with older supervisors. Ie being able to 1. fix in latest supervisor, 2. "backport"/workaround via api -> supervisor config vars has always been useful

Reply:
Added